src/game.cpp

```diff
@@ -634,11 +634,11 @@ void Game::playerMoveThing(uint32_t playerId, const Position& fromPos, uint16_t
 		}
 
 		if (movingCreature->getPosition().isInRange(player->getPosition(), 1, 1, 0)) {
-			SchedulerTask* task = createSchedulerTask(
```

Comment:
I believe in these situations we should use `auto`, because it doesn't even matter what the type is here, we just pass it along. Wdyt?

Reply:
or just move the call directly to method call

Reply:
> or just move the creation directly to method call, then we can let the compiler do its best thing

it's looks like some abomination
![Zrzut ekranu z 2024-12-27 09-49-31](https://github.com/user-attachments/assets/a0144403-f2ed-4006-b4c4-3ca2173c0701)

Reply:
std::move shouldnt be needed here, but yeah, might be true it doesnt look the best